x-pack/plugins/observability_solution/apm/server/routes/assistant_functions/get_apm_timeseries/get_transaction_failure_rate.ts

```diff
@@ -41,7 +43,7 @@ export async function getTransactionFailureRate({
       rollupInterval: RollupInterval.OneMinute,
       intervalString,
       filter: filter.concat(...termQuery(TRANSACTION_TYPE, transactionType)),
-      groupBy: 'transaction.type',
+      groupByFields: ['transaction.type', ...groupByFields],
```

Comment:
Is there any chance the `transaction.type` is already presented in the `groupByFields` array? If yes would it be a problem grouping it twice?

Reply:
Fixed in d8c2f80438305c7a2d55cbda9b10a08ea1f972cb